src/fbw/src/interface/SimConnectInterface.cpp

```diff
@@ -242,6 +242,8 @@ bool SimConnectInterface::prepareSimDataSimConnectDataDefinitions() {
   result &= addDataDefinition(hSimConnect, 0, SIMCONNECT_DATATYPE_FLOAT64, "FLAPS HANDLE INDEX", "NUMBER");
   result &= addDataDefinition(hSimConnect, 0, SIMCONNECT_DATATYPE_FLOAT64, "GEAR HANDLE POSITION", "POSITION");
 
+  result &= addDataDefinition(hSimConnect, 8, SIMCONNECT_DATATYPE_STRING32, "ATC ID", "");
```

Comment:
This should not be added here as it is not being used.

Reply:
Fixed in a new commit